DataFormats/BTauReco/interface/ShallowTagInfo.h

```diff
@@ -0,0 +1,39 @@
+#ifndef DataFormats_BTauReco_ShallowTagInfo_h
+#define DataFormats_BTauReco_ShallowTagInfo_h
+
+#include "DataFormats/BTauReco/interface/RefMacros.h"
+#include "DataFormats/BTauReco/interface/BaseTagInfo.h"
+#include "DataFormats/BTauReco/interface/CandIPTagInfo.h"
+#include "DataFormats/BTauReco/interface/CandSecondaryVertexTagInfo.h"
+#include "DataFormats/BTauReco/interface/TaggingVariable.h"
+
+namespace reco {
+
+class ShallowTagInfo : public BaseTagInfo {
+public:
+  ShallowTagInfo(void) { }
+
+  ShallowTagInfo(
+    const TaggingVariableList & list,
+    const edm::RefToBase<Jet> & jetref) :
+      list_(list),
+      jetRef_(jetref) { }
+
+  virtual ~ShallowTagInfo(void) { }
+
+  virtual ShallowTagInfo* clone(void) const { return new ShallowTagInfo(*this); }
+
+  virtual edm::RefToBase<Jet> jet(void) const { return jetRef_; }
+
+  virtual TaggingVariableList taggingVariables(void) const { return list_; }
+
+protected:
+  /*const*/ TaggingVariableList  list_;
```

Comment:
OK for debugging and "transparency" of use.
Still, this means these TagInfos can not be a part of the standard data tiers (RECO/AOD/miniAOD etc)